packages/core/helpers/ver/4.24.4.ts

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash'
```

Comment:
@adamdoe should this be 4.24.5 now?

Reply:
Yes, any new changes should go into 4.24.5.

Reply:
@Matt-DiPaolo can you make that change?